### PR TITLE
TFAC-437: Utils for logTab

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

File: graphql/database/users/__tests__/user-utils.test.js [hunk 1]
```diff
@@ -83,4 +83,158 @@ describe('user utils', () => {
...
+          .utc()
+          .subtract(5, 'days')
+          .toISOString(),
+        numTabs: 10,
+      },
+      recentDay: {
+        date: moment
+          .utc()
+          .subtract(2, 'days')
+          .toISOString(), // yesterday
```
Comment (kmjennison) [inline graphql/database/users/__tests__/user-utils.test.js:207]: Update/remove comment "yesterday".

File: graphql/database/users/__tests__/user-utils.test.js [hunk 2]
```diff
@@ -83,4 +83,158 @@ describe('user utils', () => {
...
+    const mockMaxDay = {
+      maxDay: {
+        date: moment
+          .utc()
+          .subtract(5, 'days')
+          .toISOString(),
+        numTabs: 10,
+      },
+      recentDay: {
+        date: moment.utc().toISOString(), // yesterday
```
Comment (kmjennison) [inline graphql/database/users/__tests__/user-utils.test.js:229]: Update/remove comment "yesterday".

**Review by kmjennison — APPROVED** [review]
A few outdated comments but otherwise LGTM!

**Review by amaliwanag2 — APPROVED** [review]
LGTM

**Pull request merged**